assets/src/edit-story/components/helpCenter/navigator/bottomNavigation.js

```diff
@@ -54,14 +62,37 @@ export function BottomNavigation({
   isNextDisabled,
   isPrevDisabled,
 }) {
+  // If either the prev or next has focus and become disabled,
+  // we want to force focus to the companion instead of losing
+  // it to the canvas.
+  const prevButtonRef = useRef(null);
+  const nextButtonRef = useRef(null);
+  const [local, setLocal] = useState({
+    isPrevDisabled,
+    isNextDisabled,
+  });
+  // We want to force focus one render before we disable the actual
+  // DOM elements so we can see if they're currently focused.
+  useEffect(() => {
+    [
+      isPrevDisabled && document.activeElement === prevButtonRef.current,
+      isNextDisabled && document.activeElement === nextButtonRef.current,
+    ].forEach((condition) => onCondition(condition)(forceFocusCompanion));
+    setLocal({ isPrevDisabled, isNextDisabled });
+  }, [isPrevDisabled, isNextDisabled]);
+
   return (
     <BottomNavBar
       aria-hidden={hasBottomNavigation}
       isHidden={!hasBottomNavigation}
     >
       <BottomNavButtons>
         <NavButton
-          onClick={onAllTips}
+          aria-label={__('Navigate to Help Center Main Menu', 'web-stories')}
+          onClick={() => {
+            forceFocusCompanion();
+            onAllTips();
+          }}
```

Comment:
do we need to worry about re-renders here since this arrow function gets re-created each render?

Reply:
nope! since there are really no children of the button besides the text node, causing extra re renders here is negligible